Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/NMDataServiceTest.cs

```diff
@@ -308,6 +308,36 @@ public void WhenDownloadFssFileAsyncThrowsException_ThenShouldExecuteCatch()
             Assert.IsTrue(result.IsFaulted);
         }
 
+        [Test]
+        public void WhenDownloadZipFssFileIsCalled_ThenShouldReturnByteArray()
+        {
+            string batchId = Guid.NewGuid().ToString();
+            const string fileName = "Daily 16-05-22.zip";
+
+            Stream stream = new MemoryStream(Encoding.UTF8.GetBytes("test stream"));
+
+            A.CallTo(() => _fakeAuthFssTokenProvider.GenerateADAccessToken(A<string>.Ignored));
+            A.CallTo(() => _fakefileShareService.FSSDownloadZipFile(A<string>.Ignored, A<string>.Ignored, A<string>.Ignored, A<string>.Ignored)).Returns(stream);
+
+            Task<byte[]> result = _nMDataService.DownloadZipFssFile(batchId, fileName, CorrelationId);
+
+            Assert.IsInstanceOf<Task<byte[]>>(result);
+        }
+
+        [Test]
+        public void WhenDownloadZipFssFileThrowsException_ThenShouldExecuteCatch()
```

Comment:
As per previous comment:
```suggestion
        public void WhenDownloadFSSZipFileAsyncThrowsException_ThenShouldExecuteCatch()
```